DR patterns january

klaudiarybicka opened the pull request.

Jon-Rowe-HMRC commented on src/hmrc-content-guide/tariff-stop-press-notices/index.njk:
```diff
@@ -14,6 +14,12 @@ layout: twoColumnPage.njk
+<h2 class="govuk-heading-l">When not to use</h2>
+
+<p class="govuk-body">
+  Don't update a customs information paper after it's been published, unless it's to correct a typo. A new one must be issued.
```
Please change 'Don't' to 'Do not'. Negative contractions are advised against in our style guide. We probably have this issue on other pages as just noticed on http://localhost:3000/hmrc-content-guide/customs-information-papers/ too.

Jon-Rowe-HMRC commented on src/hmrc-content-guide/customs-information-papers/index.njk:
```diff
@@ -52,17 +52,31 @@ Description (CIP #)
 Add the document to the <a href="https://www.gov.uk/government/collections/customs-information-papers--2" class="govuk-link">Customs information papers</a> collection page. Do not create a public change note.
+</p>
+
+<h2 class="govuk-heading-l">HTML attachment</h2>
```
Please make 'HTML attachment' under 'How it works' in the title structure. See http://localhost:3000/hmrc-content-guide/anti-dumping-measures/ as an example.

Jon-Rowe-HMRC reviewed (changes requested):
I've made a couple of minor in-line comments. Thanks.

klaudiarybicka commented:
I have added "Publication" medium header to comply with the new hierarchy in customs information papers, please let me know if that is fine. I also moved the "Collection page" section to the bottom as it is in other patterns.

Jon-Rowe-HMRC reviewed (approved):
Looks good to me, thanks Klaudia
